examples/boto3/README.md

```diff
@@ -26,7 +26,7 @@ aws --region=default --endpoint-url http://localhost:8000 s3api put-bucket-notif
  ```
 - Topic creation with endpoint:
 ```
-aws --endpoint-url http://localhost:8000 s3api create-topic --name=myname --attributes={nvp[0] : nvp[1]} 
+aws --endpoint-url http://localhost:8000 s3api create-topic --name=myname --attributes={'TopicArn': 'arn:aws:sns:region:acct-id:mytopic'} 
```

Comment:
topicarn is returned from the call. not passed to the call.
you need to pass attributes. take this: https://github.com/ceph/ceph/blob/master/examples/boto3/topic_with_endpoint.py#L38
is an example, but pass the attributes explictly (not via the URL-like parsing), just do:
```
--attributes{'push-endpoint': 'amqp://127.0.0.1:5672', 'amqp-exchange': 'ex1', 'amqp-ack-level': 'broker'}
```
and please test before you update the doc.

Reply:
Alright thanks.